### DISCUSSION TITLE
Proper way to access my repos from other computers?

### DISCUSSION THREAD

**Discussion by datascribbler777** [community/community#31776] [Other Feature Feedback, Questions, & Ideas]
Hello all,

I was wondering what is the proper way to work on an existing private repo from a different computer. Say I had a desktop at home that I sometimes work on but then wanted to work on the code and push back changes from a laptop?  Or maybe from my work computers.  

What is the proper way  to do this? 

I am using SSH for the initial creation of the repo. I tried adding an SSH key from another computer but when I try to clone, or push code changes after getting the zip file, git always says the repo doesn't exist. 

Just trying to figure out which tree to bark up right now. 

Maybe I should make some kind of "dummy" account and add myself as a contributor?

Any help would be greatly appreciated.

**Comment by airtower-luna** [comment]
> I am using SSH for the initial creation of the repo. I tried adding an SSH key from another computer but when I try to clone, or push code changes after getting the zip file, git always says the repo doesn't exist.

That's the right approach in principle. :slightly_smiling_face: Why it doesn't work is hard to say without knowing what exactly you did. In principle the right order is:

* Generate a key pair on the new computer (if you haven't already).
* Upload the public key to your account.
* Make sure Git can use the matching private key (e.g. that it's added to SSH-Agent, or the file in the right location).

If you're not sure why you're not recognized, run `ssh -v git@github.com` to get debug information on the SSH connection.

Also make sure you're using the SSH URL to clone, an SSH key isn't going to help you with HTTPS. :grin:
